tools/schemacode/bidsschematools/tests/test_validator.py

```diff
@@ -343,6 +343,15 @@ def test_bids_datasets(bids_examples, tmp_path):
     # Have all files been validated?
     assert len(result["path_tracking"]) == 0
 
+    # Is the schema version recorded correctly?
+    schema_path = os.path.join(
+        os.path.abspath(os.path.dirname(__file__)),
+        "../data/schema",
```

Comment:
if using `os.path` then make it 
```suggestion
        os.pardir, "data", "schema",
```
since not sure if would mix ok on windows